### PR TITLE
Feature/editing flights enabled

### PR DESCRIPTION
**Pull request by bmorris-revacomm**
- editing flights enabled
- connected send button
- fixed time bug

### CODE REVIEW

File: Logging/Controllers/Model Controllers/Form 781/Form781Controller.swift [hunk 1]
```diff
@@ -124,6 +124,25 @@ class Form781Controller {
         save()
     }
     
+    func updateFlight(flight: Flight, missionNumber: String, missionSymbol: String, fromICAO: String, toICAO: String, takeOffTime: String, landTime: String, totalTime: String, touchAndGo: String, fullStop: String, totalLandings: String, sorties: String, specialUse: String) {
```
Comment (Pete-Hoch) [inline Logging/Controllers/Model Controllers/Form 781/Form781Controller.swift:127]: Please add a unit test for this to, Form781ControllerTests.swift.

**Review by petermisik — COMMENTED** [review]
See Pete's notes.  Once that's in, I'm good.

File: Logging/Controllers/Model Controllers/Form 781/Form781Controller.swift (hunk 1 above)
Comment (bmorris-revacomm) [reply Logging/Controllers/Model Controllers/Form 781/Form781Controller.swift:127]: Done! Could you take a look at it? I haven't done unit tests before, so I more or less just copied the section from the aircrew unit test

Comment (bmorris-revacomm) [reply Logging/Controllers/Model Controllers/Form 781/Form781Controller.swift:127]: Oh just kidding, I need to fix it, hold on

Comment (bmorris-revacomm) [reply Logging/Controllers/Model Controllers/Form 781/Form781Controller.swift:127]: Fixed! Let me know if it looks okay

**Review by Pete-Hoch — APPROVED** [review]
Looks good.

**Pull request merged**